packages/postcss-ordered-values/script/buildListStyleType.js

```diff
@@ -26,15 +26,4 @@ const listTypeURL =
       }
     }
   );
-
-  writeFile(
```

Comment:
This block of code is duplicated below. It was copied to update  the file path but then the original code was not deleted, which caused a crash when trying to run the script.